src/Raven.Studio/wwwroot/App/views/common/notificationCenter/detailViewer/licenseLimitDetails.html

```diff
@@ -20,9 +20,9 @@ <h3 class="modal-title" data-bind="text: licenseLimitNotification.title"></h3>
 
         <div class="modal-footer">
             <button type="button" class="btn btn-default" data-bind="click: $root.close">Close</button>
-            <a target="_blank" href="https://ravendb.net/buy" class="btn btn-primary">
+            <a target="_blank" data-bind="click: $root.register, attr: { 'href': url }" class="btn btn-primary">
                 <i class="icon-raven"></i>
-                <span>Buy license</span>
+                <span>Request license</span>
```

Comment:
I wonder if we should divide this into 2 cases: 
a) buy some license
b) upgrade existing license 

Having just request license when user already has license is misleading

Reply:
done